perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -446,6 +446,11 @@ sub _missed_collection_reports {
     my $config = $self->feature('whitespace');
     my $state_mapping = $config->{missed_collection_state_mapping};
 
+    # Create a lookup hash mapping fms_state to text
```

Comment:
Can we assume there won't be more than one thing with the same `fms_state` (or that it'll pick one at random if there is)?

Reply:
Good point, there are lots of things for the same fms state, so this isn't going to work.